count_files/utils/file_handlers.py

```diff
@@ -167,6 +167,9 @@ def is_hidden_file_or_dir return True, even if it has visible final file/folder.
     Windows: when used separately, the function returns False if the path does not exist.
     Linux, Mac OS: when used separately, the result depends on the presence of a "/." in the path.
 
+    Windows: if filepath is a local drive it returns True.
+    is_hidden_file_or_dir('C:/') True , ...('C:') False
+    is_hidden_file_or_dir('D:/') True, ...('D:') True
```

Comment:
I don't get it. Why should the hidden status be considered different for local or non-local drives?

Reply:
I don't get it too.